### ISSUE TITLE
Serviceset CLI preview is incorrect

### ISSUE THREAD

**Issue by lazyfrosch** [Icinga/icingaweb2-module-director#2204]
I think the CLI for service sets never has been implemented correctly in the first place.

Also see #1760 

## Example

```
# icingacli director serviceset show Webserver
template ServiceSet "Webserver" {
    assign where "webserver" in host.vars.system_roles
}
```

**Comment by Tqnsls** [comment]
I'd like to add that as far as I can check the editing of a service in a serviceset (like changing vars of the service) over "icingacli director [service|serviceset] set [...]" iss not working properly, too.
Correct me if I'm wrong.

**Comment by lazyfrosch** [comment]
I think set should work, but you are only able to change stuff on the actual serviceset. Or what are you trying?

**Comment by Tqnsls** [comment]
I would for example like to add/remove/rename the included services in the service set, vars of the services in the serviceset. 
The stuff that we talked about in the director workshop :)

for example **icingacli director serviceset set "serviceset1" "service_in_serviceset1" --vars.notification_sla="never"** does add the custom var "notification_sla" with value "never" to all services in the serviceset. 
so at least the documentation on how to change stuff is lacking some information.

**Comment by dgoetz** [comment]
I would expect the CLI to handle services in a serviceset to be like it is for services at a host.

`icingacli director service set SERVICE --service_set SERVICESET`

Not sure if `service-set`, `serviceset` or `service_set` is the best option, but I think the last one fits best how other attributes are used.

**Comment by dbu1986** [comment]
The same feature request applies to Director's REST API, where it is currently not possible to list all Services in a ServiceSet, e.g. like this:

` curl -H 'Accept: application/json' -u '________:________' 'http://localhost/icingaweb2/director/serviceset/services?name=SS_Test' `

**Comment by Thomas-Gelf** [comment]
The initial request will be fixed with #2550, for related issues/feature requests please open dedicated issues.
